tree/dataframe/test/dataframe_helpers.cxx

```diff
@@ -682,3 +682,63 @@ TEST(RunGraphs, AlreadyRun)
    ROOT_EXPECT_WARNING(ROOT::RDF::RunGraphs({r1, r2, r3, r4}), "RunGraphs",
                        "Got 4 handles from which 2 link to results which are already ready.");
 }
+
+TEST(RDFHelpers, ProgressHelper_Existence_ST)
+{
+   // Redirect cout.
+   std::streambuf *oldCoutStreamBuf = std::cout.rdbuf();
+   std::ostringstream strCout;
+   std::cout.rdbuf(strCout.rdbuf());
+   auto d_write_1 = ROOT::RDataFrame(10000000).Define("x", "42").Snapshot("tree", "f1.root");
+   auto d_write_2 = ROOT::RDataFrame(10000000).Define("y", "1").Snapshot("tree", "f2.root");
+   ROOT::RDF::RNode d = ROOT::RDataFrame("tree", {"f1.root", "f2.root"});
+   ROOT::RDF::Experimental::AddProgressbar(d);
+   d.Count().GetValue();
+   // Restore old cout.
+   std::cout.rdbuf(oldCoutStreamBuf);
+   
+   std::cout << strCout.str();
+   bool is_empty = strCout.str() == "";
+   EXPECT_FALSE(is_empty);
```

Comment:
Another thing that might be bothering the CI, as seen on mac11 https://lcgapp-services.cern.ch/root-jenkins/job/root-pullrequests-build/181171/testReport/projectroot.tree.dataframe/test/gtest_tree_dataframe_test_dataframe_helpers/ 

I think we should not be printing to `stdout` since anyway we are not checking anything other than if the string is not empty. These lines should probably be changed to

```suggestion
   EXPECT_FALSE(strCout.empty());
```

And the same for all other tests.